src/SpatialLite.Core/Algorithms/Euclidean2DCalculator.cs

```diff
@@ -101,17 +101,17 @@ public double CalculateArea(ICoordinateList vertices) {
 				throw new ArgumentException("List must contain at least 3 vertices.", "vertices");
 			}
 
-			double area = 0;
-			int maxIndex = vertices.Count - 1;
-			if (vertices[0] != vertices[maxIndex]) {
-				maxIndex++;
-			}
-
-			for (int i = 0; i < maxIndex; i++) {
-				area += (vertices[(i + 1) % vertices.Count].X + vertices[i % vertices.Count].X) * (vertices[(i + 1) % vertices.Count].Y - vertices[i % vertices.Count].Y);
-			}
-
-			return area / 2.0;
+			double area = 0;
+			int maxIndex = vertices.Count - 1;
+			if (vertices[0] != vertices[maxIndex]) {
+				maxIndex++;
+			}
+
+			for (int i = 0; i < maxIndex; i++) {
+				area += (vertices[(i + 1) % vertices.Count].X + vertices[i % vertices.Count].X) * (vertices[(i + 1) % vertices.Count].Y - vertices[i % vertices.Count].Y);
+			}
+
+			return Math.Abs(area / 2.0);

```

Comment:
Area of the polygon should always be positive, is `Math.Abs` really necessary here?